webapp/components/tabs.tsx

```diff
@@ -30,7 +30,7 @@ export const Tab = function ({
   return (
     <li
       className={`
-      box-border flex h-7 items-center rounded-md px-2 py-1 text-sm font-medium
+      box-border flex h-7 items-center rounded-md px-2 py-1 mb-1 text-sm font-medium
```

Comment:
This will _always_ add a margin to each element. We better add `gap-y-1` to the `ul` component instead of this (on line 66). This way, as `flex-wrap` is already defined, the separation will appear in the `y` axis only when moving to a new line (so no margin is added to each element)